src/screens/HomeScreen.js

```diff
@@ -19,13 +19,14 @@ const HomeScreen = ({ navigation, route }) => {
     
     if (route.params?.selectedHabit) {
       const newHabit = { 
-      name: route.params.selectedHabit, 
-      completed: false, 
-      selectedAt: new Date().getTime()
+        name: route.params.selectedHabit, 
+        completed: false, 
+        selectedAt: new Date().getTime()
```

Comment:
you should start using a linter for these types of corrections, and use that linter for your code before every merge. you can search for a proper linter that works for the React Native, if there are none than search one for React

Reply:
I will install a linter for React Native.